src/utils/fluent.test.js

```diff
@@ -0,0 +1,34 @@
+import { initFluentBundles, updateAppLocale, getMessage, getRawMessage, fluentError } from './fluent.js'
+
+describe('utils:fluent', () => {
+  beforeAll(async () => {
+    await initFluentBundles()
+    updateAppLocale('en')
```

Comment:
Sorry, I just refactored this here: https://github.com/mozilla/blurts-server/pull/2720
The new function name is now `updateLocale()`, since "app" was misleading – locales are request-based not app-wide